common/src/mailbox_api.rs

```diff
@@ -140,7 +144,17 @@ impl Default for MailboxRespHeader {
 }
 
 // GET_IDEV_CSR
-// No command-specific input args
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct GetIdevCsrReq {
+    pub hdr: MailboxReqHeader,
+    pub data_size: u32,
+    pub data: [u8; GetIdevCsrReq::DATA_MAX_SIZE], // variable length
+}
```

Comment:
The idev CSR is a Certificate Signing Request generated by ROM that is self-signed. This is used during manufacturing so that the vendor's manufacturing CA can decide if they want to generate and sign a cert for the IDevID public key.

Instead of using the idev CSR command, we should define a new command (`GET_IDEV_CERT`) for for appending the signature.